sord.gemspec

```diff
@@ -24,8 +24,9 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'yard'
   spec.add_dependency 'sorbet-runtime'
-  spec.add_dependency 'commander', '~> 4.5'
-  spec.add_dependency 'parlour', '~> 5.0'
+  spec.add_dependency 'commander', '~> 5.0'
```

Comment:
This needed a bump to support more recent Ruby versions

Reply:
Commander 5.0 only supports Ruby 3 and up.
Please could you update the CI matrix in `.github/workflows` to drop Ruby 3?

This is fine imo - a breaking change, but one that Parlour made a while ago.

Reply:
Done.

BTW, would it be possible for you to either allow tests to run automatically or push up the trivial fixes yourself?  Feels like a long round trip for 6 bytes of change :).  You do have push permission on my branches.

Reply:
I've relaxed the Actions settings a bit (for both Sord and Parlour) to stop this being a pain - they must've changed at some point! Should run automatically now, but also happy to fix the tiny bits myself. Cheers

Reply:
Thanks!!